hardhat.config.js

```diff
@@ -212,7 +212,7 @@ module.exports = {
   contractSizer: {
     disambiguatePaths: false,
     runOnCompile: true,
-    strict: true,
+    strict: false,
```

Comment:
Hope, it won't find its way into upstream.

Reply:
:100:

Reply:
we are back